var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -33,19 +33,64 @@ class PyScipy(PythonPackage):
     homepage = "http://www.scipy.org/"
     url = "https://pypi.python.org/packages/source/s/scipy/scipy-0.15.0.tar.gz"
 
+    import_modules = [
+        'scipy', 'scipy._build_utils', 'scipy._lib', 'scipy.cluster',
+        'scipy.constants', 'scipy.fftpack', 'scipy.integrate',
+        'scipy.interpolate', 'scipy.io', 'scipy.linalg', 'scipy.misc',
+        'scipy.ndimage', 'scipy.odr', 'scipy.optimize', 'scipy.signal',
+        'scipy.sparse', 'scipy.spatial', 'scipy.special', 'scipy.stats',
+        'scipy.weave', 'scipy.io.arff', 'scipy.io.harwell_boeing',
+        'scipy.io.matlab', 'scipy.optimize._lsq', 'scipy.sparse.csgraph',
+        'scipy.sparse.linalg', 'scipy.sparse.linalg.dsolve',
+        'scipy.sparse.linalg.eigen', 'scipy.sparse.linalg.isolve',
+        'scipy.sparse.linalg.eigen.arpack', 'scipy.sparse.linalg.eigen.lobpcg',
+        'scipy.special._precompute'
+    ]
+
     version('0.18.1', '5fb5fb7ccb113ab3a039702b6c2f3327',
             url="https://pypi.python.org/packages/22/41/b1538a75309ae4913cdbbdc8d1cc54cae6d37981d2759532c1aa37a41121/scipy-0.18.1.tar.gz")
     version('0.17.0', '5ff2971e1ce90e762c59d2cd84837224')
     version('0.15.1', 'be56cd8e60591d6332aac792a5880110')
     version('0.15.0', '639112f077f0aeb6d80718dc5019dc7a')
 
     depends_on('python@2.6:2.8,3.2:')
-    depends_on('py-nose', type='build')
-    # Known not to work with 2.23, 2.25
-    depends_on('binutils@2.26:', type='build')
+    depends_on('py-setuptools', type='build')
     depends_on('py-numpy@1.7.1:+blas+lapack', type=('build', 'run'))
 
+    # FIXME: Is this really necessary?
+    # Known not to work with 2.23, 2.25
+    # depends_on('binutils@2.26:', type='build')
```

Comment:
We've discussed this before, but things like `binutils` and `curl` aren't really dependencies of any particular package. They are necessary for every package in Spack. In a way, they are Spack dependencies. The only reason people have been adding them is that certain packages require newer versions of these dependencies than are found on their ancient systems. We added a [note about this](http://spack.readthedocs.io/en/latest/getting_started.html#binutils) in the documentation. I think this dependency should be removed. It won't even build on macOS.